hybrid-nodes-cdk/lib/nodeadm-stack.ts

```diff
@@ -218,7 +220,6 @@ export class NodeadmBuildStack extends cdk.Stack {
           actions: [
             'iam:DeleteRolePolicy',
             'iam:DetachRolePolicy',
```

Comment:
Do we still need the `DetachRolePolicy` permission here if we're adding it in the block above?